Solutions/TenableIO/Data Connectors/TenableIO.json

```diff
@@ -0,0 +1,156 @@
+	"additionalRequirementBanner": "These queries and workbooks are dependent on a [**Tenable.io parser**](https://aka.ms/sentinel-tenableio-parser) based on Kusto to work as expected which is deployed with the Azure Sentinel Solution.",
+	"graphQueries": [
+		{
+			"metricName": "Total data received",
+			"legend": "Tenable_IO_Assets_CL",
+			"baseQuery": "Tenable_IO_Assets_CL"
+		},
+		{
+			"metricName": "Total data received",
+			"legend": "Tenable_IO_Vulns_CL",
+			"baseQuery": "Tenable_IO_Vulns_CL"
+		}
+	],
+	"sampleQueries": [
+		{
+			"description": "TenableIO VM Report - All Assets",
+			"query": "Tenable_IO_Assets_CL\n | sort by TimeGenerated desc"
+		},
+		{
+			"description": "TenableIO VM Report - All Vulns",
+			"query": "Tenable_IO_Vulns_CL\n | sort by TimeGenerated desc"
+		},
+		{
+			"description": "Select unique vulnerabilities by a specific asset.",
+			"query": "Tenable_IO_Vuln_CL\n | where asset_fqdn_s has \"one.one.one.one\"\n | summarize any(asset_fqdn_s, plugin_id_d, plugin_cve_s) by plugin_id_d"
+		},
+		{
+			"description": "Select all Azure assets.",
+			"query": "Tenable_IO_Assets_CL\n | where isnotempty(azure_resource_id_s) or isnotempty(azure_vm_id_g)"
+		}
+	],
+	"dataTypes": [
+		{
+			"name": "Tenable_IO_Assets_CL",
+			"lastDataReceivedQuery": "Tenable_IO_Assets_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+		},
+		{
+			"name": "Tenable_IO_Vulns_CL",
+			"lastDataReceivedQuery": "Tenable_IO_Vulns_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+		}
+	],
+	"connectivityCriterias": [
+		{
+			"type": "IsConnectedQuery",
+			"value": [
+				"Tenable_IO_Assets_CL\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+			]
+		},
+		{
+			"type": "IsConnectedQuery",
+			"value": [
+				"Tenable_IO_Vulns_CL\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+			]
+		}
+	],
+	"availability": {
+		"status": 1,
+		"isPreview": true
+	},
+	"permissions": {
+		"resourceProvider": [
+			{
+				"provider": "Microsoft.OperationalInsights/workspaces",
+				"permissionsDisplayText": "read and write permissions are required.",
+				"providerDisplayName": "Workspace",
+				"scope": "Workspace",
+				"requiredPermissions": {
+					"write": true,
+					"read": true,
+					"delete": true
+				}
+			},
+			{
+				"provider": "Microsoft.OperationalInsights/workspaces/sharedKeys",
+				"permissionsDisplayText": "read permissions to shared keys for the workspace are required. [See the documentation to learn more about workspace keys](https://docs.microsoft.com/azure/azure-monitor/platform/agent-windows#obtain-workspace-id-and-key).",
```

Comment:
PR validation is not passing because permissionsDisplayText is not matching with Template. According to template it should be  **"permissionsDisplayText": "read and write permissions are required.",**

Reply:
Fixed